core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -20,8 +20,16 @@ oppia.constant(
   'STORE_PLAYTHROUGH_URL',
   '/explorehandler/store_playthrough/<exploration_id>');
 
+oppia.constant('PLAYTHROUGH_ENABLED_EXPLORATION_WHITELIST', [
+  // NOTE: These are the exploration ids of the Fractions lessons.
+  "umPkwp0L1M0-", "MjZzEVOG47_1", "9trAQhj6uUC2", "rfX8jNkPnA-1",
+  "0FBWxCE5egOw", "670bU6d9JGBh", "aHikhPlxYgOH", "-tMgcP1i_4au",
+  "zW39GLG_BdN2", "Xa3B_io-2WI5", "6Q6IyIDkjpYC", "osw1m5Q3jK41"
+]);
+
 oppia.factory('PlaythroughService', [
-  '$http', 'LearnerActionObjectFactory', 'PlaythroughObjectFactory',
+  '$http', 'LearnerActionObjectFactory',
+  'PlaythroughObjectFactory',
```

Comment:
Not sure why the line break was added, but make sure that the line breaks here match with those in line 42 onwards.

Reply:
Done.